Retranslated Task.RunSynchronously Method

terasato opened the pull request.

opbld32 commented:
The PR is not being built because the target branch `live` is filtered in `.openpublishing.publish.config.json` on `live` branch.

opbld32 commented:
The PR is not being built because the target branch `live` is filtered in `.openpublishing.publish.config.json` on `live` branch.

opbld33 commented:
The PR is not being built because the target branch `live` is filtered in `.openpublishing.publish.config.json` on `live` branch.

terasato commented:
Hi, I rebased this branch onto the current `live` branch.
I've removed unnecessary lines, and removed tailing spaces.
I hope this will help your merging. Thanks.

v-radelg commented:
Hi @terasato thank you for remove the lines and spaces, but we are afraid something went wrong and now apparently there are infinite files changed, making this PR not possible to work on. 
Sadly we will have to close it, but please feel free to create a new PR if the file still need modifications.

Thanks for your time and support.

Kind regards, Microsoft C&E Localization Team

opbld33 commented:
The PR is not being built because the target branch `live` is filtered in `.openpublishing.publish.config.json` on `live` branch.

terasato commented:
Something must have gone wrong. I created a new PR.
